zazu/repo/commands.py

```diff
@@ -1,5 +1,7 @@
 # -*- coding: utf-8 -*-
 import click
+from git import Git
```

Comment:
all other imports in zazu use the `import foo` syntax rather than the `from foo import bar` syntax.